user/enterprise/trusty.md

```diff
@@ -41,8 +32,10 @@ Next, run the following to complete the install:
 
 `sudo bash /tmp/installer.sh --travis_enterprise_host="[travis.yourhost.com]" --travis_enterprise_security_token="[RabbitMQ Password/Enterprise Security Token]"`
 
+This installer uses Docker's `aufs` storage driver. If this default doesn't work out for you, please [get in touch with us](mailto: enterprise@travis-ci.com?subject=Trusty%20Workers) to discuss alternatives.
+
 There are two differences with the non-AWS installer. First, it uses AUFS instead of DeviceMapper and, second doesn't have strict storage device layout requirements which would otherwise be required for setting up the DeviceMapper volumes.
```

Comment:
> non-AWS installer
I guess now it technically _is_ the AWS installer

Reply:
Ahh - thanks so much for catching that! That paragraph was meant to be deleted 😓